lib/uniris/oracles/transaction_content.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Uniris.Oracles.TransactionContent do
+  @moduledoc false
+
+  defstruct [
+    :mfa,
+    :payload
+  ]
+
+  def serialize(%__MODULE__{mfa: {m, f, a}, payload: payload}) do
+    m_binary = :erlang.atom_to_binary(m)
+    f_binary = :erlang.atom_to_binary(f)
```

Comment:
Why do you want to serialize the function name ? As we should have the knowledge based on the Oracle type the function to call (like behavior)